config/requirements/base.pip

```diff
@@ -23,6 +23,7 @@ mimeparse==0.1.3
 mock==2.0.0
 jsonfield==1.0.3
 psycopg2==2.6.2
+fake-factory==0.7.4
```

Comment:
Instead of adding the fake-factory dependency, you can set the factory_boy version to 2.8.1